examples/all-clusters-app/nrfconnect/main/AppTask.cpp

```diff
@@ -36,6 +36,11 @@
 #include <credentials/DeviceAttestationCredsProvider.h>
 #include <credentials/examples/DeviceAttestationCredsExample.h>
 
+#ifdef CONFIG_WIFI_NRF700X
```

Comment:
Are those changes needed given that we're going to use NCS samples, anyway? If we prefer to keep it, maybe we should stick to the same samples (lock-app and light-switch) as in NCS?

Reply:
Hmm yes, that's doubtful. AFAIK we aim to support all-clusters-app in the upstream in the future. But you are right, we may keep this repo ncs inclined for now. Any other suggestions, @kkasperczyk-no, @LuDuda?

Reply:
Agree, I would limit to the changes necessary for NCS. The changes to upstream examples should be somehow synced with the changes to samples in NCS, so let's wait for merging them to NCS first.

Reply:
Ok, done.